core/src/test/java/com/ibm/wala/core/tests/cha/JunitEntrypointSupportTest.java

```diff
@@ -25,12 +25,18 @@ public void basic() throws ClassHierarchyException, IllegalArgumentException, IO
     ClassHierarchy cha = ClassHierarchyFactory.make(scope);
     List<Entrypoint> entrypoints =
         Iterator2Collection.toList(JUnitEntryPoints.make(cha).iterator());
-    assertEquals(2, entrypoints.size());
+    assertEquals(4, entrypoints.size());
     assertEquals(
-        "< Application, Ljunit/JunitTests, test1()V >([<Application,Ljunit/JunitTests>])",
+        "< Application, Ljunit/JunitTests$A, test2()V >([<Application,Ljunit/JunitTests$A>])",
```

Comment:
For readability can we switch these assertions to use the Hamcrest matcher `assertThat` and `contains` methods?   You can see an example [here](https://github.com/wala/WALA/blob/df5caa49a8c1ce5592ae98f996da2d609f278f8f/util/src/test/java/com/ibm/wala/util/test/BasicGraphTest.java).  `contains` is varargs and checks that the list contents exactly equal the arguments in order.  We can then also get rid of the check on `entrypoints.size()`

Reply:
> We can then also get rid of the check on `entrypoints.size()`

Don't we want to know if there are other entry points? There could be erroneous ones.